types/react-router-redux/index.d.ts

```diff
@@ -25,15 +25,15 @@ export interface ConnectedRouterProps<State> {
 }
 export class ConnectedRouter<State> extends React.Component<ConnectedRouterProps<State>> {}
 
-export const LOCATION_CHANGE: string;
+export const LOCATION_CHANGE: '@@router/LOCATION_CHANGE';
```

Comment:
I'll disagree. IMO, it's not good to export values from type definition files. What's the benefit of it?